specification/src/main/asciidoc/webprofile/Introduction.adoc

```diff
@@ -70,30 +69,29 @@ core (“à la carte” installation).
 
 *
 
-As dictated by the general rules for Java EE
+As dictated by the general rules for Jakarta EE
 profiles in the Platform specification, products that implement the Web
 Profile must honor:
 
-. all requirements of the Java EE Platform
+. all requirements of the Jakarta EE Platform
 specification that apply to all profiles;
 . all requirements of this specification;
 . all requirements of the individual
 component specifications;
-. all requirements in the Java EE Platform
+. all requirements in the Jakarta EE Platform
 specification that are conditional on the presence of a specific
 technology or combinations of technologies.
 
 Let’s look at some examples of requirements
 from each grouping.
 
-For the first one, the Java EE Platform
-specification mandates support for the “ _java:_ ” naming context in all
+For the first one, the Jakarta EE Platform
+specification mandates support for the “ _jakarta:_ ” naming context in all
```

Comment:
This talks about Java SE. Should be kept untouched.

Reply:
No, this is about Java EE, so should be changed to "Jakarta EE",
but the `java:` JNDI namespace is not being changed in Jakarta EE 8.